python/lsst/ip/diffim/subtractImages.py

```diff
@@ -69,12 +72,26 @@ class SubtractInputConnections(lsst.pipe.base.PipelineTaskConnections,
         dimensions=("instrument", "visit"),
         storageClass="ExposureCatalog",
         name="finalVisitSummary",
+        # TODO: remove on DM-39854.
+        deprecated=(
+            "Deprecated in favor of visitSummary.  Will be removed after v26."
+        )
+    )
+    visitSummary = connectionTypes.Input(
```

Comment:
For consistency, please either name this `calibrationsCatalog` in the connections, or use `visitSummary` later instead of `calibrationsCatalog`